src/components/DiseaseTable.tsx

```diff
@@ -103,6 +105,29 @@ const DiseaseTable = ({
         ),
     };
 
+    const handleCellInHoveredRow = (
```

Comment:
this may be me, so feel free to disregard, but in your attempt to make this name really clear, I find it kinda hard to read. I think `handleCell` or `handleCellInteraction` or `onCell` or `onCellInteraction` would all be easier to read

Reply:
Haha, makes me laugh because I was trying to be extra explicit in your footsteps, thanks for feedback. I think I like `onCellInteraction` but definitely open minded here.